util/rlp-compress/src/common.rs

```diff
@@ -1,4 +1,4 @@
-// Copyright 2015-2017 Parity Technologies
+// Copyright 2015-2019 Parity Technologies
```

Comment:
Not caused by this PR, but:

```suggestion
// Copyright 2015-2019 Parity Technologies
// Copyright 2015-2019 Parity Technologies (UK) Ltd.
```

That's probably the reason why previous PR missed this. Same for below.